src/components/AddMeeting.js

```diff
@@ -0,0 +1,99 @@
+import React from "react"
+import { useState, useEffect } from "react"
+import { useParams } from "react-router-dom"
+import { useNavigate } from "react-router-dom"
+
+export default function AddMeeting() {
+  const navigate = useNavigate()
+  const params = useParams()
+  const [date, setDate] = useState("")
```

Comment:
I like the way u structure your code, but If u use an object in the state instead, it's more efficient, shorter and more commonly used in the tech industry